components/Snackbar/src/MDCSnackbarManager.m

```diff
@@ -486,6 +485,15 @@ - (void)removeSuspensionIdentifierMainThread:(NSUUID *)identifier forCategory:(N
 
 @implementation MDCSnackbarManager
 
+static UIColor *_snackbarMessageViewBackgroundColor;
+static UIColor *_snackbarMessageViewShadowColor;
+static UIColor *_messageTextColor;
+static UIFont *_messageFont;
+static UIFont *_buttonFont;
+static NSMutableDictionary<NSNumber *, UIColor *> *_buttonTitleColors;
+static BOOL _mdc_adjustsFontForContentSizeCategory;
+static BOOL _applyStylingOnCurrentSnackbar;
```

Comment:
Can these live in `MDCSnackbarManagerInternal sharedInstance`?

Reply:
yep!